README.md

```diff
@@ -21,23 +21,22 @@
 * [Collaborators](#collaborators)
   * [Moderator](#moderator)
   * [Co-Moderators](#co-moderators)
-* [Guidelines](#guidelines)
-  * [Contributing](#contributing)
+* [Contributing Guidelines](#contributing)
```

Comment:
Can you fix this link? 
Doesnt seem to redirect.

Reply:
Sure. Let me check.